external-report-demo/report-demo/app.js

```diff
@@ -0,0 +1,93 @@
+const crypto = require('crypto');
+const queryString = require('query-string');
+
+let response;
+
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ *
+ */
+exports.lambdaHandler = async (event, context) => {
+    try {
+      const json = validateJSON(event);
+
+      const renderCSV = (key) => {
+        if (json[key] && json[key].length > 0) {
+          message = Object.keys(json[key][0]).join(",") + "\n";
+          message += json[key].map(item => Object.values(item).join(",")).join("\n");
```

Comment:
Do we need to worry about escaping special characters like `,` or `"` here?

Reply:
As far as I can tell, this is not needed from what the portal would be handing us as the query.